CONTRIBUTING.md

```diff
@@ -1,76 +1,118 @@
+your time.
+
+This repository contains the files for the
+[Fluent Bit documentation library](https://docs.fluentbit.io/). Keeping these docs
+separate from the [main Fluent Bit repository](https://github.com/fluent/fluent-bit)
+helps reduce the number of commits to the Fluent Bit source code and makes it
+easier to maintain both projects.
+
+Fluent Bit has a group of dedicated maintainers who oversee this repository,
+including several technical writers. These writers will review any pull requests
+you open, so don't be afraid to contribute—even if you're not a writer by trade,
+your suggestions are valuable, and we'll help you wrangle any stray commas.
+
+## GitBook
+
+The Fluent Bit docs library is built and hosted through
+[GitBook](https://docs.gitbook.com/). Unfortunately, GitBook doesn't support
+local previews for contributors, but a Fluent Bit maintainer with a dedicated GitBook
+account can verify that things are formatted correctly after you open a new pull
+request.
+
+Each `.md` file in this repository is a single page. You can use
+[standard Markdown syntax](https://docs.gitbook.com/content-editor/editing-content/markdown)
+to edit existing pages, or create a new `.md` file to add an additional page to
+the docs library. If you create a new page, you'll also need to update
+[GitBook's `SUMMARY.md` file](https://docs.gitbook.com/integrations/git-sync/content-configuration#structure)
+(or ask a maintainer to update it for you).
 
 ## Workflow
 
-All documentation contributions arrives as Pull Requests (PR) on Github in the repository [fluent/fluent-bit-docs](https://github.com/fluent/fluent-bit-docs). Then some of the maintainers of Fluent Bit will review it, triage it, add comments if needed, or merge it.
-
-Once a PR is merged, a third-party service called [Gitbook](https://gitbook.com) will receive a notification and will grab the latest changes, render a new site and update the content of [docs.fluentbit.io](https://docs.fluentbit.io).
-
-## Source Code Structure
+After you open a pull request in this repo, a Fluent Bit maintainer will review
+it, triage it, add comments or suggestions as needed, and then merge it. After
+your changes are successfully merged into `master`, the docs site will update
+within a few minutes.
 
-Documentation source code structure depends on Fluent Bit source code structure and its versions. In Fluent Bit source code, we  have a stable branch and a development branch; as of now, these are:
+### Stale pull requests
 
-- stable branch: [1.8](https://github.com/fluent/fluent-bit/tree/1.8)
-- development branch: [master](https://github.com/fluent/fluent-bit/tree/master)
+If you open a pull request that requires ongoing discussion or review, the
+Fluent Bit maintainers will add a [`waiting-for-user` tag](#tags) to your pull
+request. This tag means that we're blocked from moving forward until you reply;
+to keep contributions from going stale, we'll wait [three weeks?] for your response,
```

Comment:
```suggestion
request. This tag means that we're blocked from moving forward until you reply.
To keep contributions from going stale, we'll wait [three weeks?] for your response,
```

You'll want to get that three weeks more concrete though, we don't want to merge it with a guess.

Reply:
yep, good call. per our Slack discussion I'm gonna bump this to six weeks (but in a separate commit so I can modify a few more lines at once)